joint_values:
  -
    - -0.05232859030365944
    - -0.3233131766319275
    - 0.3114327192306519
    - 1.732351541519165
    - 0.1577998548746109
    - 0.3321604430675507
    - 0.3498024940490723
  -
    - -0.03813019394874573
    - -0.271706759929657
    - 0.3112016916275024
    - 1.850474953651428
    - 0.1181069612503052
    - 0.5517301559448242
    - 0.3753677010536194
  -
    - -0.2595849335193634
    - -0.2679450511932373
    - 0.1996336281299591
    - 1.918121099472046
    - 0.1485399752855301
    - 0.561043381690979
    - 0.02705126442015171
  -
    - -0.5026637315750122
    - -0.3867769837379456
    - 0.1411070227622986
    - 1.747105598449707
    - 0.3761958181858063
    - 0.3860852420330048
    - -0.3586363792419434
  -
    - -0.6888631582260132
    - -0.57383131980896
    - 0.1179292351007462
    - 1.531747937202454
    - 0.5291157364845276
    - 0.1550414860248566
    - -0.5726268887519836
  -
    - -0.5805776715278625
    - -1.009967565536499
    - 0.09980244189500809
    - 0.7308778166770935
    - 0.4020991027355194
    - -0.4504794776439667
    - -0.3715276718139648
  -
    - -0.4260375797748566
    - -0.775666356086731
    - 0.1135288029909134
    - 0.9447458386421204
    - 0.327337771654129
    - -0.3388741016387939
    - -0.2563818991184235
  -
    - -0.2310882657766342
    - -0.6128501892089844
    - 0.1231072470545769
    - 1.079471468925476
    - 0.1888568997383118
    - -0.2542611360549927
    - -0.1084932163357735
  -
    - -0.02844025380909443
    - -0.5252817273139954
    - 0.1287579834461212
    - 1.191470861434937
    - -0.03194421157240868
    - -0.1882148087024689
    - 0.03886328265070915
  -
    - 0.1231296136975288
    - -0.5053938031196594
    - 0.1582280844449997
    - 1.238555431365967
    - -0.1194460242986679
    - -0.179716020822525
    - 0.1976438909769058
  -
    - 0.2011633217334747
    - -0.4995000660419464
    - 0.1307977885007858
    - 1.310388565063477
    - -0.3274876177310944
    - -0.1720772087574005
    - 0.2211080491542816
  -
    - 0.2863125205039978
    - -0.5217810869216919
    - 0.1689207255840302
    - 1.332957744598389
    - -0.2970702946186066
    - -0.1602446287870407
    - 0.3510779142379761
  -
    - 0.3396822214126587
    - -0.5301966667175293
    - 0.146503672003746
    - 1.3685222864151
    - -0.4338150024414062
    - -0.1861696392297745
    - 0.3675599992275238
  -
    - 0.4209449887275696
    - -0.5815184116363525
    - 0.1792632639408112
    - 1.374976873397827
    - -0.3983151316642761
    - -0.170802116394043
    - 0.4928953349590302
  -
    - 0.372498095035553
    - -0.3055878579616547
    - 0.2462988793849945
    - 1.809605598449707
    - -0.4876466393470764
    - 0.1790888905525208
    - 0.5824191570281982
  -
    - 0.2397503852844238
    - -0.2343670576810837
    - 0.2184786647558212
    - 1.77146315574646
    - -0.3347142040729523
    - 0.2167636156082153
    - 0.3631178140640259
  -
    - -0.1132349595427513
    - -0.3353443145751953
    - 0.07401327788829803
    - 1.505643606185913
    - 0.1119977980852127
    - 0.1192115694284439
    - -0.237152561545372
  -
    - -0.2949992716312408
    - -0.5372291207313538
    - 0.0159895196557045
    - 1.211210131645203
    - 0.2697331607341766
    - -0.08934565633535385
    - -0.4544923305511475
  -
    - -0.5139084458351135
    - -0.9325181841850281
    - 0.005762138869613409
    - 0.6398502588272095
    - 0.4234155714511871
    - -0.4491605460643768
    - -0.5596210956573486
  -
    - -0.4556738436222076
    - -0.8312963247299194
    - 0.008384407497942448
    - 1.020215153694153
    - 0.4117193222045898
    - -0.1692401170730591
    - -0.6205191016197205
  -
    - -0.2347392737865448
    - -0.5136540532112122
    - -0.004173524677753448
    - 1.410550117492676
    - 0.2259245961904526
    - 0.1222460940480232
    - -0.4592427015304565
  -
    - -0.04226715490221977
    - -0.3583285808563232
    - 0.00954485684633255
    - 1.711868047714233
    - -0.003660448361188173
    - 0.380787581205368
    - -0.2142796814441681
  -
    - 0.1360613852739334
    - -0.3016414642333984
    - -0.002119037555530667
    - 1.949864983558655
    - -0.3518487811088562
    - 0.5514799356460571
    - 0.1378487944602966
  -
    - 0.01445069722831249
    - 0.01226692646741867
    - 0.002525125397369266
    - 2.449119806289673
    - -0.5254948139190674
    - 1.096468687057495
    - 0.2074371576309204
  -
    - -0.03528859838843346
    - -0.02356395684182644
    - -0.1104963272809982
    - 2.151411533355713
    - 0.2188538908958435
    - 0.7991206049919128
    - -0.485365241765976
  -
    - -0.2418825030326843
    - -0.3318041563034058
    - -0.2620086967945099
    - 1.728952646255493
    - 0.4477790892124176
    - 0.5016424655914307
    - -0.9144209623336792
  -
    - -0.3628061413764954
    - -0.2008762508630753
    - -0.2732406556606293
    - 1.967375040054321
    - 0.6111209988594055
    - 0.6328670382499695
    - -1.183377623558044
  -
    - -0.4448740184307098
    - -0.06473866105079651
    - -0.3328161537647247
    - 2.035701990127563
    - 0.5505217909812927
    - 0.527588963508606
    - -1.255447030067444
  -
    - -0.4574733674526215
    - -0.34267657995224
    - 0.1481554210186005
    - 1.53973400592804
    - 0.1162293180823326
    - 0.2095771133899689
    - -0.2231171429157257
  -
    - -0.4231575131416321
    - -0.1877694129943848
    - 0.1237906441092491
    - 1.717612385749817
    - 0.1457395851612091
    - 0.2359213531017303
    - 0.2314707785844803
  -
    - -0.5950608253479004
    - -0.2077496945858002
    - 0.04658946022391319
    - 1.684338331222534
    - 0.3684300780296326
    - 0.1796304732561111
    - -0.05224461853504181
  -
    - 0.08243624120950699
    - -0.4728094637393951
    - 0.1441692560911179
    - 1.266333222389221
    - -0.2467212378978729
    - -0.1991787105798721
    - 0.7563959360122681